luigi/parameter.py

```diff
@@ -370,11 +370,12 @@ class DateHourParameter(Parameter):
     19:00.
     """
 
-    date_format = '%Y-%m-%dT%H'  # ISO 8601 is to use 'T'
+    date_format = '%Y-%m-%dT%H'
```

Comment:
I didn't see the point of the comment and since I removed the one below, I thought I'd remove it here as well. ISO 8601 is being mentioned above anyways, so it doesn't add much IMO. Also, I think it's rather arbitrary to document `T` and not `YYYY-MM-DD`, for instance.

I can undo the change on this line if you prefer :blush: